proto/hummock.proto

```diff
@@ -28,6 +28,7 @@ message SstableInfo {
   uint64 uncompressed_file_size = 11;
   uint64 range_tombstone_count = 12;
   BloomFilterType bloom_filter_kind = 13;
+  map<uint32, common.Buffer> vnode_bitmap_mapping = 14;
```

Comment:
We should make this field `optional` so that we can distinguish between SST written by old codes or SST written by new codes with empty vnode bitmap.

Reply:
Should we deprecate `table_ids`? What happen to this mapping when split happens?

Reply:
unfortunately, optional not allowed on map type